graphistry/umap_utils.py

```diff
@@ -287,7 +287,7 @@ def _bundle_embedding(self, emb, index):
             emb = pd.DataFrame(emb, columns=[config.X, config.Y], index=index)
         elif emb.shape[1] == 2 and 'cudf.core.dataframe' in str(getmodule(emb)):
             import cudf
-            emb = cudf.DataFrame(emb.to_cupy(), columns=[config.X, config.Y], index=index.to_cupy())
+            emb = cudf.DataFrame(emb.values, columns=[config.X, config.Y], index=index.values)
```

Comment:
what is `emb` -- isn't it a `cudf.DataFrame` already? this might just be a `emb.rename(columns={..})`? I'm not sure of what/why here

Reply:
emb is float values so it needs to be made into df

Reply:
very strange... getmodule does say it is a cudf df, but rename does not work on emb since its a cupy
following works but not pretty enough im guessing:
emb.columns=[config.X, config.Y]
emb.index=index

Reply:
what do `isinstance(emb, cudf.DataFrame)` and `type(emb)` say?